### ISSUE TITLE
Test failures for matching gift and partial soft credit in multi currency enabled orgs.

### ISSUE THREAD

**Issue by njjc** [SalesforceFoundation/Cumulus#2210]
```
System.SObjectException: SObject row was retrieved via SOQL without querying the requested field: Opportunity.CurrencyIsoCode
```
in two places:
```
MTCH_FindGifts_CTRL.oppTotalSelected
PSC_ManageSoftCredits_CTRL.oppTotalSoftCredit
```

Both of these are expecting the CurrencyIsoCode to exist on the opportunity, but the query for that opportunity isn't selecting it properly.

**Comment by njjc** [comment]
**lurch: add

**Comment by LurchTheButler** [comment]
Tracking <a href="https://foundation.my.salesforce.com/a2x80000003g633AAA" target="_blank">W-011016</a>

**Comment by mrbelvedere** [comment]
Included in beta release 3.80 (Beta 8)

**Comment by mrbelvedere** [comment]
Included in production release rel/3.80
